### DISCUSSION TITLE
Please allow us to set the initial state of collapsed resource groups on first load

### DISCUSSION THREAD

**Discussion by vesper8** [laravel/nova-issues#4601] [Ideas & Feature Requests]
I have a huge amount of resources and resource groups and certain administrators have requested that some resource groups always be collapsed on page load since they are hardly ever used.

I was previously accomplishing this using https://github.com/dcasia/collapsible-resource-manager but this no longer supports Nova v4 because Nova has implemented the menu-building features. However it seems you do not allow us to set the initial state, furthermore you force on us the "ability to remember collapsed state" which is actually unwanted in my use case.

So, I would like to request that you please allow us to hard-code the initial collapsed state using a true/false boolean, and if possible let us also disable this feature that makes the browser remember the collapsed state.

Many thanks!

**Comment by jhoff** [comment]
Additionally, a global default collapsed state would be nice.

Also the ability to collapse all / none on a resource perhaps?
